src/float.rs

```diff
@@ -777,6 +781,11 @@ pub trait FloatCore: Num + NumCast + Neg<Output = Self> + PartialOrd + Copy {
     /// Returns the mantissa, base 2 exponent, and sign as integers, respectively.
     /// The original number can be recovered by `sign * mantissa * 2 ^ exponent`.
     ///
+    /// This function is implemented for `f128`, however, lossily. The mantissa part
+    /// of `f128` is 113 bits, which does not fit in 64 bits. This function truncates
+    /// the radix to its 64 most significant bits, and the original number can be
+    /// estimated by `sign * mantissa * 2 ^ 49 * 2 ^ exponent`.
```

Comment:
Can't we add that 49 into the returned exponent too?

Reply:
Fixed.